src/view/drag-handle/drag-handle.jsx

```diff
@@ -45,6 +45,10 @@ export default class DragHandle extends Component<Props> {
     [canLiftContextKey]: PropTypes.func.isRequired,
   };
 
+  static defaultProps = {
+    canCancelDragWithForceTouch: true,
```

Comment:
this should be a standard prop for a drag handle an an optional prop on the Draggable (with a defaultProp)